orca-queue-redis/src/main/kotlin/com/netflix/spinnaker/orca/q/redis/RedisQueue.kt

```diff
@@ -229,6 +244,13 @@ class RedisQueue(
       } else {
         try {
           val message = convertToMessage(json, mapper)
+
+          // TODO: AttemptsAttribute could replace `attemptsKey`
+          message.setAttribute(
```

Comment:
I think we can unify but the initial rollout likely needs to keep them separate because of old/new messages.

After we're new-only then I reckon `RedisQueue.retry()` can just check the message attribute.